CRM/Financial/BAO/FinancialType.php

```diff
@@ -246,6 +246,42 @@ public static function permissionedFinancialTypes(&$permissions, $descriptions)
   }
 
   /**
+   * CRM-21026
+   * Wrapper aroung getAvaliableFinancialTypes to get all including disabled FinancialTypes
+   * @param array $financialTypes
+   *   an array of financial types
+   * @param int|string $action
+   *   the type of action, can be add, view, edit, delete
+   * @param bool $resetCache
+   *   load values from static cache
+   *
+   * @return array
+   */
+  public static function getAllAvailableFinancialTypes($financialTypes = NULL, $action = CRM_Core_Action::VIEW, $resetCache = FALSE) {
+    // Flush pseudoconstant cache
+    CRM_Contribute_PseudoConstant::flush('financialType');
+    $fTypes = self::getAvailableFinancialTypes($financialTypes, $action, $resetCache, TRUE);
+    return $fTypes;
+  }
+
+  /**
+   * CRM-21026
```

Comment:
I don't think we need the CRM here - if we do it should be after the main descriptor